### ISSUE TITLE
Uncaught Error: No line exists for row 17. Last screen row: 16

### ISSUE THREAD

**Issue by uncedric** [atom/atom#5943]
[Enter steps to reproduce below:]

1. CTRL + p  add
2. CTRL + p commit

**Atom Version**: 0.186.0
**System**: Ubuntu 14.04
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: No line exists for row 17. Last screen row: 16

```
At /usr/share/atom/resources/app/src/text-editor-presenter.js:385

Error: No line exists for row 17. Last screen row: 16
  at TextEditorPresenter.<anonymous> (/usr/share/atom/resources/app/src/text-editor-presenter.js:385:19)
  at TextEditorPresenter.module.exports.TextEditorPresenter.batch (/usr/share/atom/resources/app/src/text-editor-presenter.js:91:12)
  at TextEditorPresenter.module.exports.TextEditorPresenter.updateLinesState (/usr/share/atom/resources/app/src/text-editor-presenter.js:375:19)
  at TextEditorPresenter.module.exports.TextEditorPresenter.getState (/usr/share/atom/resources/app/src/text-editor-presenter.js:123:14)
  at TextEditorComponent.module.exports.TextEditorComponent.updateSync (/usr/share/atom/resources/app/src/text-editor-component.js:172:38)
  at /usr/share/atom/resources/app/src/text-editor-component.js:280:28
  at ViewRegistry.module.exports.ViewRegistry.performDocumentUpdate (/usr/share/atom/resources/app/src/view-registry.js:159:9)
  at /usr/share/atom/resources/app/src/view-registry.js:3:61

```

### Commands

```
     -1:48.6 core:backspace (atom-text-editor.editor.mini)
     -1:45.5 core:confirm (atom-text-editor.editor.mini)
  3x -1:07.4 window:toggle-full-screen (atom-text-editor.editor.is-focused)
     -1:03.3 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:50.2 core:confirm (atom-text-editor.editor.mini)
     -0:50.2 git-plus:add (atom-text-editor.editor.is-focused)
     -0:40.9 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:38.4 core:confirm (atom-text-editor.editor.mini)
     -0:38.4 git-plus:commit (atom-text-editor.editor.is-focused)
     -0:30.8 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:30.6 core:backspace (atom-text-editor.editor.is-focused)
     -0:27.2 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:24.6 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
  2x -0:24.1 core:move-left (atom-text-editor.editor.is-focused)
     -0:18.5 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:11.6 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      "node_modules",
      "bower_components",
      "min.js",
      "min.css",
      ".coffee",
      ".svn",
      ".png",
      ".jpg"
    ],
    "disabledPackages": [
      "archive-view",
      "wrap-guide",
      "feedback",
      "metrics",
      "release-notes",
      "status-bar",
      "Zen",
      "background-tips",
      "language-c",
      "language-java",
      "language-make",
      "language-objective-c",
      "language-perl",
      "language-todo",
      "language-toml",
      "language-yaml",
      "link",
      "open-on-github",
      "package-generator",
      "spell-check",
      "styleguide",
      "symbols-view",
      "language-go",
      "language-property-list",
      "language-ruby",
      "language-ruby-on-rails",
      "deprecation-cop",
      "keybinding-resolver",
      "welcome",
      "project-ring",
      "Clean",
      "autosave",
      "language-mustache",
      "language-sass",
      "markdown-preview",
      "bookmarks",
      "encoding-selector",
      "language-coffee-script",
      "ask-stack",
      "language-xml"
    ],
    "themes": [
      "seti-ui",
      "atom-dark-syntax"
    ],
    "autoHideMenuBar": true
  },
  "editor": {
    "showIndentGuide": true,
    "softWrap": true,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v1.4.4
css-snippets, v0.5.0
git-plus, v4.4.8
highlight-line, v0.9.3
html-id-class-snippets, v1.4.1
javascript-snippets, v1.0.0
language-ejs, v0.1.0
linter, v0.12.0
linter-jshint, v0.1.0
minimap, v4.5.0
project-manager, v1.15.5
seti-ui, v0.5.1

# Dev
No dev packages
```

**Comment by uncedric** [comment]
I am going to close this issue until I got some tome to provide all the details/images info listed on the guidelines

**Comment by izuzak** [comment]
@uncedric This should be fixed in Atom 0.187.0 (see https://github.com/atom/atom/pull/5904). If you manage to reproduce in 0.187.0 -- please let us know.
